README.md

```diff
@@ -1,4 +1,20 @@
 [![CircleCI](https://circleci.com/gh/giantswarm/k8sportforward.svg?&style=shield&circle-token=a1a6d79291b52022182456a68ac420b8c793344a)](https://circleci.com/gh/giantswarm/k8sportforward)
 
 # k8sportforward
+
 Package `k8sportforward` exposes functionality similar to `kubectl port-forward`.
+
+
+
+## Projects using k8sportforward
+
+### Test libraries
+
+### Libraries
+
+- https://github.com/giantswarm/apprclient
+- https://github.com/giantswarm/helmclient
+
+### Services & operators
+
+- https://github.com/giantswarm/chart-operator
```

Comment:
I just encountered problems with updating vendor of apprclient. I encourage everybody to create such PRs. That would make vendoring more seamless.